tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -481,4 +481,31 @@ class CompletionMatchSuite extends BaseCompletionSuite {
                 |}""".stripMargin
     ),
   )
+  check(
+    "exhaustive-map".tag(IgnoreScala2),
```

Comment:
Scala 2 could use that completions too :)

Reply:
I am working on bigger refactor of Scala 2 match/case completions, like adding pattern only completions, and I will include this change there too :)